# Assignment 6

To measure the psychological state of Americans, a psychology researcher (in 1929) devised
three questions put to 3443 Americans from different walks of life. He asked them whether they
were happy. They could answer only **yes** or **no** to each question. By today’s standards the study
is quite simple, and he did no formal statistical analysis of his data. The frequency data (number
of **yes** and **no** responses within each employment category) from this study are in the file
*Americandata.csv*.

---

You need to use the data and answer the question, that is, **Is the type of profession related to the happiness of American people?**
Report the **χ2** statistic and the *statistical significance of test*, in addition to the contingency table.
Also, explain the results and what they mean. What does the value of **χ2** statistic mean? Is it
good or bad? Is the test statistically significant?

In [ ]:
#code here
import pandas as pd
from scipy.stats import chi2_contingency

# Load the data
data = pd.read_csv("./Americandata.csv")

print("Data:")
print(data)

# Create contingency table
table = data.set_index(data.columns[0])

print("\nContingency Table:")
print(table)

# Chi-square test
chi2, p, dof, expected = chi2_contingency(table)

print("\nChi-Square Test Results:")
print("Chi-square statistic:", chi2)
print("Degrees of freedom:", dof)
print("p-value:", p)

if p < 0.05:
    print("Result: Statistically significant")
else:
    print("Result: Not statistically significant")

Data:
  Happy  Students  Laborers  Preachers  Physicians  Housewives  Teachers  \
0   Yes       390       378         35         159          78       108   
1    No      1610       122        265          51         122        38   

   Lawyers  Musicians  
0       11         31  
1       64         19  

Contingency Table:
       Students  Laborers  Preachers  Physicians  Housewives  Teachers  \
Happy                                                                    
Yes         390       378         35         159          78       108   
No         1610       122        265          51         122        38   

       Lawyers  Musicians  
Happy                      
Yes         11         31  
No          64         19  

Chi-Square Test Results:
Chi-square statistic: 936.1394782304601
Degrees of freedom: 7
p-value: 7.523435335099551e-198
Result: Statistically significant


## Analysis
The contingency table shows how many people in each profession answered yes or no when asked if they were happy. The responses were different across the professions. For example, most laborers and physicians answered yes, while most students, preachers, and lawyers answered no.

The chi-square test gave a x² value of 936.139, with 7 degrees of freedom and a p-value of 7.523 × 10⁻¹⁹⁸. Since the p-value is much smaller than 0.05, the results are statistically significant. This means there is a strong enough difference in the responses to conclude that profession and happiness are related in this study.

The x² value shows how different the actual results are from what we would expect if profession had nothing to do with happiness. A value of 936.139 is very large, meaning the differences between the professions are much greater than we would expect by chance. The value is not necessarily good or bad; it is just used to measure how much the results differ from what would be expected if there were no relationship.

Overall, the results suggest that the type of profession is related to the happiness of the Americans in this study. However, this does not mean that having a certain profession causes someone to be happy or unhappy. It only shows that there is a relationship between the two variables.
